src/DesignQuery/scripts/test.tcl

```diff
@@ -0,0 +1,2 @@
+puts "Hello from Boston, It's me Zaid !"
```

Comment:
What's this test for ? We have a separate location for tcl tests.

Reply:
Removed